packages/ui-common/src/util/keyring.ts

```diff
@@ -20,17 +21,17 @@ function keyringHelper (type: AddressType, keyring: Keyring, address?: string):
   const keyringFunction = (type === AddressType.Account ? keyring.getAccount : keyring.getAddress).bind(keyring);
 
   return fromOption(new Error('You need to specify an address'))(fromNullable(address))
-    // `keyring.getAddress` might fail: catch and return None if it does
+    // `keyring.getAddress` might fail: catch and return None if it does, pass on the KeyringAddress otherwise
```

Comment:
I think there might be a way to do with `fromNullable` of fp-ts.

But that's alright, we can leave it like this also,